pyiceberg/io/pyarrow.py

```diff
@@ -1099,21 +1099,70 @@ def map_value_partner(self, partner_map: Optional[pa.Array]) -> Optional[pa.Arra
         return partner_map.items if isinstance(partner_map, pa.MapArray) else None
 
 
-_PRIMITIVE_TO_PHYSICAL = {
-    BooleanType(): "BOOLEAN",
-    IntegerType(): "INT32",
-    LongType(): "INT64",
-    FloatType(): "FLOAT",
-    DoubleType(): "DOUBLE",
-    DateType(): "INT32",
-    TimeType(): "INT64",
-    TimestampType(): "INT64",
-    TimestamptzType(): "INT64",
-    StringType(): "BYTE_ARRAY",
-    UUIDType(): "FIXED_LEN_BYTE_ARRAY",
-    BinaryType(): "BYTE_ARRAY",
-}
-_PHYSICAL_TYPES = set(_PRIMITIVE_TO_PHYSICAL.values()).union({"INT96"})
+def _primitive_to_phyisical(iceberg_type: PrimitiveType) -> str:
+    return visit(iceberg_type, _PRIMITIVE_TO_PHYISCAL_TYPE_VISITOR)
+
+
+class PrimitiveToPyhsicalType(SchemaVisitorPerPrimitiveType[str]):
```

Comment:
Typo: `Pyhsical` (unless maybe that's how it's spelled in the python community)

Reply:
🤣  Good old typo